src/components/timing/TimingControls.vue

```diff
@@ -103,30 +96,49 @@ const handleStop = async () => {
               :menu-icon="ArrowDown"
               :rules="[rules.required]"
             ></v-select>
-            <p v-else>
-              <v-btn class="ma-0 pa-0" variant="plain" color="black" :to="{ name: 'projects' }"
-                >Add a new project</v-btn
-              >
-            </p>
+
+            <!--Button appears when no projects exist-->
+            <PrimaryButton
+              v-else
+              class="mb-5 w-50 align-self-center"
+              color="tertiary"
+              text="Add new Project"
+              :to="{ name: 'projects' }"
+            ></PrimaryButton>
+            <!--This button only appears when there are projects in existence-->
+            <PrimaryButton
+              v-if="projectsStorage.currentCustomerProjects.length != 0"
```

Comment:
** This is not working!! **  Now the button is rendered when the length >= 1. 
** It must be rendered when length is equal to 0. **

1. When evaluating conditions, try to use 3 operators if possible.

`Instead of " != ", use " !== "`


2. Following the idiomatic.js principles:
```
// When only evaluating that an array is empty,
// instead of this:
if ( array.length === 0 ) ...

// ...evaluate truthiness, like this:
if ( !array.length ) ...
```